packages/amplication-server/src/core/app/app.service.ts

```diff
@@ -51,6 +54,12 @@ export class AppService {
         },
         roles: {
           create: USER_APP_ROLE
+        },
+        environments: {
+          create: {
+            name: DEFAULT_ENVIRONMENT_NAME,
+            address: cuid()
```

Comment:
Please source from an environment variable or add a todo comment.

Reply:
The address only represents the subdomain and it must be globally unique. By default the address is based on a CUID but the users have an option to choose a custom sub domain later.
As you move forward with the rest of the deployment code, if you see a reason to include the full domain name in the address, we can come back to this code.

Reply:
Oh... I see. Okay, we'll come back to this later.